src/core/sharedTypes.ts

```diff
@@ -0,0 +1,11 @@
+export type Button = {
```

Comment:
`sharedTypes` is a odd name file. I will probably just move these types into the other type file and rename Button into `FrameButtonMetadata` so it's more explicit.

Reply:
Moved the types into the existing `farcasterTypes.ts` file and renamed `Button` to `FrameButtonMetadata` as suggested.